packages/admin-sdk/src/in-app-purchases/index.ts

```diff
@@ -0,0 +1,14 @@
+import { createSender } from '../channel';
+
+/**
+ * Trigger the in-app purchase flow modal.
+ */
+export const trigger = createSender('inAppPurchases');
+
+export type inAppPurchases = {
+    responseType: unknown,
+
+    featureName: string,
+
+    priceModel: string,
```

Comment:
aren't we handling this now for the merchant? The selection of the price model? Then we don't need this. May we add it later as an optional string (though actual it's an integer id in the SBP).